lib/tasks/advisory.rake

```diff
@@ -0,0 +1,54 @@
+namespace :advisory do
+  public def remove_unaffected(unaffected_versions)
+    version_array = []
+    unaffected_versions.each do |versions|
+      version_array << if versions.split.length > 2
+                         versions.split(',')
+                       else
+                         versions
+                       end
+    end
+    version_array.flatten!
+    version_array.each do |versions|
+      unaffectedv_array = versions.split
+      unaffected_version_number = unaffectedv_array.second
+      case unaffectedv_array.first
+      when ">="
+        delete_if { |v| Gem::Version.new(v) >= Gem::Version.new(unaffected_version_number) }
+      when "<="
+        delete_if { |v| Gem::Version.new(v) <= Gem::Version.new(unaffected_version_number) }
+      when "~>"
+        delete_if { |v| Gem::Version.new(v) == Gem::Version.new(unaffected_version_number) }
+      when ">"
+        delete_if { |v| Gem::Version.new(v) > Gem::Version.new(unaffected_version_number) }
+      when "<"
+        delete_if { |v| Gem::Version.new(v) < Gem::Version.new(unaffected_version_number) }
+      end
+    end
+  end
+
+  desc 'Mark versions as vulnerable'
+  task update: :environment do
+    begin
+      system "git submodule update"
+      vuln = {}
+      count = 0
+      Dir.chdir(Rails.root.join('storage', 'ruby-advisory-db'))
+      ActiveRecord::Base.transaction do
+        Dir.glob("**/*.yml") do |gem_name|
```

Comment:
This would also pick yml files from [rubies directory](https://github.com/rubysec/ruby-advisory-db/tree/master/rubies).

Reply:
We don't want those, right? I'll remove them and iterate on only the [gems](https://github.com/rubysec/ruby-advisory-db/tree/master/gems) directory.

Reply:
yes, we are only concerned with gems dir.